apollo-ios/Sources/Apollo/HTTPResponse.swift

```diff
@@ -12,22 +12,31 @@ public class HTTPResponse<Operation: GraphQLOperation> {
   /// The raw data received from the URL loading system
   public var rawData: Data
   
-  /// [optional] The data as parsed into a `GraphQLResult`, which can eventually be returned to the UI. Will be nil if not yet parsed.
+  /// [optional] The data as parsed into a `GraphQLResult`, which can eventually be returned to the UI. Will be nil 
+  /// if not yet parsed.
   public var parsedResponse: GraphQLResult<Operation.Data>?
   
-  /// [optional] The data as parsed into a `GraphQLResponse` for legacy caching purposes. If you're not using the `JSONResponseParsingInterceptor`, you probably shouldn't be using this property.
+  /// [optional] The data as parsed into a `GraphQLResponse` for legacy caching purposes. If you're not using the 
+  /// `JSONResponseParsingInterceptor`, you probably shouldn't be using this property.
   /// **NOTE:** This property will be removed when the transition to the Swift Codegen is complete.
```

Comment:
We should remove this `**NOTE**` since we've don't that transition and now are only not removing it for backwards compatibility.

Reply:
Ah, good catch!